### PR TITLE
Point Sdk AAR updated as per 15.3.0

### PR DESCRIPTION
**Pull request by nehabluedot**

### CODE REVIEW

**Review by jsprincep — APPROVED** [review]
Nice work! 🎉

**Review by jsprincep — CHANGES_REQUESTED** [review]
As per discussion, I think we need to add our location dependencies to the wrapper to satisfy: https://bluedotinnovation.atlassian.net/browse/BD-2013

**Comment by nehabluedot** [comment]
> As per discussion, I think we need to add our location dependencies to the wrapper to satisfy: https://bluedotinnovation.atlassian.net/browse/BD-2013

location dependency added

**Comment by jsprincep** [comment]
I notice the minimal app still has the location dependencies? Have you tested that the sdk can still be used if the location dependencies are removed from the app?

**Comment by nehabluedot** [comment]
> I notice the minimal app still has the location dependencies? Have you tested that the sdk can still be used if the location dependencies are removed from the app?
As I remember it requires location dependency to successfully build the app, although build errors were not from the SDK.

File: BDPointXamarinBinder.csproj [hunk 1]
```diff
@@ -261,6 +234,7 @@
     <EmbeddedReferenceJar Include="Jars\lifecycle-common-java8-2.2.0.jar" />
     <EmbeddedReferenceJar Include="Jars\converter-moshi-2.7.1.jar" />
     <EmbeddedReferenceJar Include="Jars\retrofit-2.7.1.jar" />
+    <EmbeddedReferenceJar Include="Jars\kotlin-stdlib-1.4.30-RC.jar" />
```
Comment (jsprincep) [inline BDPointXamarinBinder.csproj:237]: Why do we need 1.4.30?

Comment (nehabluedot) [reply BDPointXamarinBinder.csproj:237]: Xamarin packages provide Kotlin.Stdlib v1.4.20 and there were few workmanager kotlin missing classes resulting in crash while running the geo trigger.
So added 1.4.30 to fix it

**Review by jsprincep — APPROVED** [review]

**Pull request merged**